### DISCUSSION TITLE
some question abont client function

### DISCUSSION THREAD

**Discussion by zafir-zhong** [rustdesk/rustdesk#767] [Q&A] [answered]
function maybe Abnormal of client?
client: mac , Android(huawei mate40pro) 1.1.9
server: docker rustdesk/rustdesk-server:latest 
1. both can not unlock or lock remote screen
2. connect failed when mobile lock after a minutes
3. connect successful when mac lock, but only a black screen

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/550
